.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
+max-line-length = 99
+select: E,W,F,C,N
+exclude:
+  venv
+  .git
+  build
+  dist
+  *.egg_info
```

Comment:
This can be dropped and placed in the `pyproject.toml`

Reply:
Thanks Jon. My mistake (I forgot to delete this file). Now it's fixed (file deleted and content present in `pyproject.toml`).